home-service/src/main/resources/service.properties

```diff
@@ -1,6 +1,6 @@
 #Path to text
-path.invitation.registration="template/invitation-to-registration.mustache"
-path.invitation.cooperation="template/invitation-to-cooperation.mustache"
-path.invitation.apartment="template/invitation-to-apartment.mustache"
+path.invitation.registration=home-service/src/main/resources/template/invitation-to-registration.mustache
```

Comment:
Do we really need it? @likeRewca Please check it out.

Reply:
I accidentally added these changes and forgot to remove them before committing